it/uk/gov/hmrc/helpers/controllers/EtmpCheckControllerSpec.scala

```diff
@@ -17,309 +17,104 @@
-        |}
-        |]}
-      """.stripMargin
+    "return 204" when {
 
-    val successResponse: JsValue = Json.parse(
-      s"""{"processingDate":"2015-12-17T09:30:47Z","etmpFormBundleNumber":"123456789012345","awrsRegistrationNumber": "DummyRef"}"""
-    )
+      "EMAC feature flag is false" in {
+        FeatureSwitch.disable(AWRSFeatureSwitches.regimeCheck())
+        stubAuditPosts
 
-    FeatureSwitch.enable(AWRSFeatureSwitches.regimeCheck())
+        val resp: WSResponse = await(client(controllerUrl).post(checkEtmpPostDataInd))
+        resp.status mustBe 204
+      }
 
-    stubbedGet(s"""$regimeURI""", OK, jsResultString)
-    stubbedPost(s"""$baseURI$subscriptionURI$safeId""", BAD_REQUEST, successResponse.toString)
-    stubbedPut(s"/enrolment-store-proxy/enrolment-store/enrolments/$enrolmentKey", OK)
+      "EMAC feature flag is true" when {
 
-    val controllerUrl = routes.EtmpCheckController.checkEtmp().url
+        "there is a business partner and regime in ETMP" when {
 
-    val resp: WSResponse = await(client(controllerUrl).post(jsonPostData))
-    resp.status mustBe 204
-  }
-
-  "return 200" when {
-
-    "AWRS feature flag is true" in {
+          "the status of the AWRS subscription is a NonSelfHealStatus" in {
 
-      val jsResultString =
-        """
-          |{
-          |  "sapNumber": "1234567890",
-          |  "safeId": "XE0001234567890",
-          |  "agentReferenceNumber": "JARN1234567",
-          |  "regimeIdentifiers": [
-          |    {
-          |      "regimeName": "AWRS",
-          |      "regimeRefNumber": "XAAW00000123456"
-          |    }
-          |  ],
-          |  "nonUKIdentification": {
-          |    "idNumber": "123456",
-          |    "issuingInstitution": "France Institution",
-          |    "issuingCountryCode": "FR"
-          |  },
-          |  "isEditable": true,
-          |  "isAnAgent": false,
-          |  "isAnIndividual": false,
-          |  "organisation": {
-          |    "organisationName": "sdfsdf",
-          |    "isAGroup": false,
-          |    "organisationType": "Corporate body"
-          |  },
-          |  "addressDetails": {
-          |    "addressLine1": "100 SomeStreet",
-          |    "addressLine2": "Wokingham",
-          |    "addressLine3": "Surrey",
-          |    "addressLine4": "London",
-          |    "postalCode": "DH14EJ",
-          |    "countryCode": "GB"
-          |  },
-          |  "contactDetails": {
-          |    "regimeName": "AWRS",
-          |    "phoneNumber": "01332752856",
-          |    "mobileNumber": "07782565326",
-          |    "faxNumber": "01332754256",
-          |    "emailAddress": "stephen@manncorpone.co.uk"
-          |  }
-          |}
-        """.stripMargin
+            FeatureSwitch.enable(AWRSFeatureSwitches.regimeCheck())
+            stubAuditPosts
+            stubEtmpRegistrationResponse(OK, etmpRegistrationResultInd)
+            stubShowAndRedirectExternalCalls("Individual")
```

Comment:
Should this be Organisation for a business partner scenario?

Reply:
Scratch this - I can see that we are using Individual integrationData from the it utils